commit 82f51372143f0025b883bbf1398e03265a0a1af7 (refs/changes/06/5206/4)
Author: Rui Loura <rui@joyent.com>
Date:   2019-01-03T19:51:05+00:00 (9 months ago)
    
    TRITON-1023 sdc-sdc doc should use network names
    Reviewed by: Robert Bogart <Robert.bogart@joyent.com>
    Approved by: Robert Bogart <Robert.bogart@joyent.com>

diff --git a/docs/index.md b/docs/index.md
index 18d3c02..e6d44db 100644
--- a/docs/index.md
+++ b/docs/index.md
@@ -190,7 +190,7 @@ for COAL by running the following from your Mac:
 
     sdc-vmapi /vms/$(vmadm lookup -1 alias=sdc0)?action=add_nics -X POST -d@- <<EOP | sdc sdc-waitforjob
     {
-        "networks": [{"primary": true, "uuid": "$(sdc-napi /networks?name=external | json -H 0.uuid)"}]
+        "networks": [{"primary": true, "name": "external"}]
     }
     EOP
     sleep 10  # wait for the sdc zone to reboot
